File: PubeoAPI/Model/Localite.cs
using System;
using System.Collections.Generic;
using System.ComponentModel.DataAnnotations;

namespace PubeoAPI.model {
    public class Localite 
    {
        public Localite(){
            Particuliers = new HashSet<Particulier>();
            Professionnels = new HashSet<Professionnel>();
        }
        [Key]
        public String CodePostal { get; set; }
        public String Ville { get; set; }
        public virtual ICollection<Particulier> Particuliers { get; set; }
        public virtual ICollection<Professionnel> Professionnels { get; set; }
    }
}

File: PubeoApp/app/src/main/java/com/example/pubeo/model/Localite.java
package com.example.pubeo.model;

public class Localite {

    private String codePostal;
    private String ville;

    public Localite(String codePostal, String ville) {
        this.codePostal = codePostal;
        this.ville = ville;
    }

    public String getCodePostal() {
        return codePostal;
    }

    public String getVille() {
        return ville;
    }
}
